spec/support/matchers/be_a_user_representation.rb

```diff
@@ -1,12 +1,12 @@
-RSpec::Matchers.define :be_a_user_representation do |user|
+RSpec::Matchers.define :be_a_user_representation do
   match do |json|
-    response_attributes = user.sliced_attributes %w(
+    response_attributes = %w(
       id
       authentication_token
       email
     )
 
     expect(json).to be
-    expect(json).to include_attributes(response_attributes)
```

Comment:
`include_attributes` matcher converts an entire hash to JSON string and tries to find it in the given JSON. It seems oddly. Instead of this we can simply check set of keys.